dags/scrape_factory.py

```diff
@@ -86,13 +86,12 @@ def previous_scrape_done(**kwargs):
             check_previous = ShortCircuitOperator(
                 task_id='check_previous',
                 python_callable=previous_scrape_done,
-                provide_context=True
             )
 
             scrape = BlackboxDockerOperator(
```

Comment:
Docker operators aren't working anymore. We'll have to figure out why before deploying for testing.